pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -1296,6 +1355,17 @@ func afwGetReleventStatefulsetByKind(ctx context.Context, cr splcommon.MetaObjec
 	return sts
 }
 
+// getPremiumAppScopePlaybookContext returns the idxc playbook context
```

Comment:
nit: getPremiumAppScopePlaybookContext returns the premium apps scope playbook context